extract_endpoint/tests/test_azure_utils.py

```diff
@@ -0,0 +1,70 @@
+import io
+import pytest
+from azure.storage import blob
+import azure_utils
+
+
+@pytest.fixture
+def sample_account() -> str:
+    return 'devstoreaccount1'
+
+
+@pytest.fixture
+def sample_key() -> str:
+    return 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
+
+
+@pytest.fixture
+def sample_container() -> str:
+    return 'energuide-extracted-data'
+
+
+@pytest.fixture
+def sample_domain() -> str:
+    return 'http://127.0.0.1:10000/devstoreaccount1'
+
+
+@pytest.fixture
+def sample_storage_coordinates(sample_account: str,
+                               sample_key: str,
+                               sample_container: str,
+                               sample_domain: str) -> azure_utils.StorageCoordinates:
+    return azure_utils.StorageCoordinates(account=sample_account, key=sample_key,
+                                          container=sample_container, domain=sample_domain)
```

Comment:
Do we need the above 4 fixtures, or can we get rid of them by putting these constants directly in this line? Generally, they should only be separate fixtures if something else also needs them separately from the `StorageCoordinates` fixture.